components/table_kv/src/obkv.rs

```diff
@@ -488,6 +500,33 @@ impl TableKv for ObkvImpl {
 
         Ok(values.remove(VALUE_COLUMN_NAME).map(Value::as_bytes))
     }
+
+    fn delete(&self, table_name: &str, key: &[u8]) -> std::result::Result<i64, Self::Error> {
+        let size = self
+            .client
+            .delete(table_name, bytes_to_values(key))
+            .context(DeleteData { table_name })?;
+
+        Ok(size)
+    }
+
+    fn batch_delete(
+        &self,
+        table_name: &str,
+        key_list: Vec<Vec<u8>>,
+    ) -> std::result::Result<i64, Self::Error> {
+        let mut batch_ops = ObTableBatchOperation::default();
```

Comment:
I guess we can pre-allocate capacity for the ObTableBatchOperation.

Reply:
Yes, I will update it.